.github/workflows/tests.yml

```diff
@@ -10,10 +10,10 @@ jobs:
         runs-on: ubuntu-20.04
         steps:
             - uses: actions/checkout@v2
-            - name: Use Node.js 16
+            - name: Use Node.js 18
               uses: actions/setup-node@v2
               with:
-                  node-version: 16.x
+                  node-version: 18.x
```

Comment:
Thoughts on nuking the hard coded version, both in the name and [pointing to the `.nvmrc` file](https://github.com/actions/setup-node/blob/869f4dd0c7f320ae834c2724d92a364de3893c24/docs/advanced-usage.md#node-version-file)?
```
- uses: actions/setup-node@v3
  with:
    node-version-file: '.nvmrc'
```

Reply:
oh nice, I didn't know you can do that ❤️